src/storage/txn/sched_pool.rs

```diff
@@ -102,94 +183,99 @@ impl SchedPool {
                     tls_flush(&reporter);
                 })
         };
-        // FIXME: for performance issue, disable priority pool temporarily
-        // if let Some(ref r) = resource_ctl {
-        //     SchedPool::Priority {
-        //         worker_pool: builder(pool_size, "sched-worker-pool")
-        //             .build_priority_future_pool(r.clone()),
-        //         resource_ctl: r.clone(),
-        //     }
-        // } else {
-        SchedPool::Vanilla {
+        let vanilla = VanillaQueue {
             worker_pool: builder(pool_size, "sched-worker-pool").build_future_pool(),
-            high_worker_pool: builder(std::cmp::max(1, pool_size / 2), "sched-high-pri-pool")
+            high_worker_pool: builder(std::cmp::max(1, pool_size / 2), "sched-worker-high")
                 .build_future_pool(),
+        };
+        let priority = resource_ctl.as_ref().map(|r| PriorityQueue {
+            worker_pool: builder(pool_size, "sched-worker-priority")
+                .build_priority_future_pool(r.clone()),
+            resource_ctl: r.clone(),
+        });
+        let queue_type = if resource_ctl.is_some() {
+            info!("sched-worker pool use dynamic mode");
+            QueueType::Dynamic
+        } else {
+            info!("sched-worker pool use vanilla mode");
+            QueueType::Vanilla
+        };
```

Comment:
```suggestion
        let queue_type = if resource_ctl.is_some() {
            QueueType::Dynamic
        } else {
            QueueType::Vanilla
        };
        info!("new sched-worker pool"; "type" => ?QueueType);
```
I think this log is not very useful, maybe just remove it.